testdata/simple/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Slightly more complex: the proper way to test that we have things set up is to just compile and run a small program that makes calls to C (think autoconf's helper programs).  We could check for individual binaries, but it's not necessarily a good indicator that cgo will work.